sources/javascripts/concat/global/main.js

```diff
@@ -0,0 +1,220 @@
+        resultsContainer: $('.js-voog-search-modal-inner').get(0),
+        // Defines if modal should close on sideclick.
+        sideclick: true,
+        // Mobile checkpoint
+        mobileModeWidth: 480,
+        // Updates results on every keypress.
+        updateOnKeypress: true,
+        // String for feedback if no results are found.
+        noResults: noResultsString
+      });
+    }
+  };
+
+  // Function to limit the rate at which a function can fire.
+  var debounce = function (func, wait, immediate) {
+    var timeout;
+    return function () {
+      var context = this, args = arguments;
+      var later = function () {
+        timeout = null;
+        if (!immediate) func.apply(context, args);
+      };
+      var callNow = immediate && !timeout;
+      clearTimeout(timeout);
+      timeout = setTimeout(later, wait);
+      if (callNow) func.apply(context, args);
+    };
+  };
+
+  var bindSideClicks = function () {
+    $('.container').on('mousedown', function (event) {
+      if (!$(event.target).closest('.js-prevent-sideclick').length) {
+        $('.js-popover').removeClass('expanded');
+        $('.js-search-close-btn').trigger('click');
+        $('.js-image-settings-popover').toggleClass('active');
+      };
+    });
+  };
+
+  // Switches the site language to the selected value from the language menu.
+  var handleLanguageSwitch = function () {
+    $('.menu-lang').find('.menu').change(function () {
+      window.location = $(this).find(':selected').val();
+    });
+  };
+
+  // Shows/hides the popover main menu (visible on smalles screens).
+  var toggleMainMenu = function () {
+    $('.js-menu-btn').click(function () {
+      $(this).toggleClass('open');
+      $('.js-menu-main').toggleClass('expanded');
+    });
+  };
+
+  // Removes optional content elements if element doesn't have any content.
+  var removeOptionalContent = function () {
+    optionalContent = $('.js-content-optional');
+    $.each($(optionalContent), function () {
+      optionalContentLength = $(this).text().trim().length;
+
+      if (!optionalContentLength > 0) {
+        $(this).remove();
+      }
+    });
+  };
+
+  // Scrolls to the comment-form if comment submit failed (to show the error messages to the user)
+  var focusFormWithErrors = function () {
+    $(document).ready(function () {
+      if ($('.comment-form').hasClass('form_with_errors')) {
+        $('html, body').scrollTop($('.comment-form').offset().top);
+      } else if ($('form').find('.form_error, .form_notice').length > 0) {
+        $('html, body').scrollTop($('.form_error, .form_notice').closest('form').offset().top);
+      }
+    });
+  };
+
+  var toggleImageSettingsPopover = function () {
+    $('.js-image-settings-popover').toggleClass('active');
+  }
+
+  $('.js-toggle-image-settings').click(function () {
+    toggleImageSettingsPopover();
+  });
+
+  // Initiates the table horisontal scroll function when window is resized.
```

Comment:
Alates sellest reast, kuni `init` funktsioonini liigutada lõõpu `// Enables the usage of the initiations outside this file.` rea ette.